airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -24,6 +24,7 @@
 
 class StripeStream(HttpStream, ABC):
     url_base = "https://api.stripe.com/v1/"
+    api_version = "2022-08-01"
```

Comment:
Is this not working for the latest version `2022-11-15`? I found it [here](https://stripe.com/docs/api/versioning)

Reply:
Tried for the version we have set in our test account. Will try to update it to latest version and check